src/ErrorPage.php

```diff
@@ -80,6 +80,15 @@ class ErrorPage extends Page
      */
     private static $store_filepath = null;
 
+    /**
+     * An array of error codes to allow in the CMS
+     * If unset, defaults to all available codes (see self::getCodes())
+     *
+     * @config
+     * @var ?array
+     */
+    private static $allowed_error_codes = null;
```

Comment:
```suggestion
     * If null, defaults to all available codes (see self::getCodes())
     */
    private static ?array $allowed_error_codes = null;
```
- `@config` is implied, so it's not needed here.
- We prefer to explicitly typehint new properties, even config properties, instead of using phpdoc. The typehint itself isn't enforced by the config system but could be in a future release.

Reply:
Ah ok, so I should treat changes like a self-contained piece of work that adheres to best practices, rather than trying to keep it consistent with the existing file. Got it 😉 will keep that in mind for future PRs